### DISCUSSION TITLE
Can a 4.x client installed on WinXP connect to a 5.x server?

### DISCUSSION THREAD

**Discussion by PizzaProgram** [SoftEtherVPN/SoftEtherVPN#1774] [General]
I'm planing to host a 5.02 server to serve 100+ HUBs (ca. 300 windows PCs) on a VPS.

- Is 5.02 linux build stable enough now for long therm usage?
- Can a 4.x client connect to it from a Windows XP? 
 _(or are there any limitations, like manually accepting cert...)_
- Why isn't there any windows builds of 5.02? 
 _(Only of 5.01 which is 2020-04-30 = 3y. old)_

Thank you in forward for any clarification.

**Comment by shakibamoshiri** [comment]
> Is 5.02 linux build stable enough now for long therm usage?  

you should test and check what part you need to use - to me it is not

> Can a 4.x client connect to it from a Windows XP?  

Yes

> Why isn't there any windows builds of 5.02?  

I think they planned to do so

**Reply by shipitsin-ilia** [reply]
there're windows builds in Azure Pipeline artifacts.
but you are correct, there's no official build yet. we should emit soon, as there are many changes
